### DISCUSSION TITLE
Doesn't work on chrome ver 102.0.5005.185 (Official Build) (64-bit)?

### DISCUSSION THREAD

**Discussion by ethkoth** [3kh0/ext-remover#259] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
My school chromebook is on chrome os ver 102.0.5005.185 (Official Build) (64-bit) but for some reason whenever I try to use ingot or ltbeef it just resets the toggle and doesn't work whenever i leave the sites. Is it possible my school just full blocked ingot and ltbeef? I know it doesn't work on ver 106 but did 102 also patch it?

**Comment by pl4udible** [comment, marked as answer]
I think they have also patched it aswell, or you are doing an error in the processing of the bookmarklets
A new version will be uploaded soon so we have t be patient

**Reply by ILikeBypassing** [reply]
hes on version 102

**Reply by ethkoth** [reply]
i dug through the discussions for this repos and i found that they patched on ver 102. Thanks!

**Comment by ILikeBypassing** [comment]
can you record a video of that happening, I may be able to figure out the problem
other than that, it could be that your policy: chrome://policy might have "javascript:" blocked which makes bookmarklets work
